.buildkite/target/pipeline.yml

```diff
@@ -45,3 +45,13 @@ steps:
         agents:
           slurm_mem: 8GB
           slurm_gpus: 1
+
+      - label: ":tree: Land"
+        command: "julia --color=yes --project=.buildkite experiments/benchmarks/land.jl"
```

Comment:
I don't think "land" is a very descriptive name here - maybe we could use "soil_canopy.jl" ?

Reply:
changed!